src/rpc/blockchain.cpp

```diff
@@ -2694,9 +2694,7 @@ UniValue CreateUTXOSnapshot(
     result.pushKV("base_height", tip->nHeight);
     result.pushKV("path", path.u8string());
     result.pushKV("txoutset_hash", maybe_stats->hashSerialized.ToString());
-    // Cast required because univalue doesn't have serialization specified for
-    // `unsigned int`, nChainTx's type.
-    result.pushKV("nchaintx", uint64_t{tip->nChainTx});
+    result.pushKV("nchaintx", tip->nChainTx);
```

Comment:
faa90f6e7b6b8c531e1ae142a5c2f568b48502a9: was this always wrong or is there some recent change to UniValue?